tests/selenium/pixel_tracking_test.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+
+import unittest
+import pbtest
+
+class pixelTrackingTesting(pbtest.PBSeleniumTest):
+	"""Tests for the cookie pixel tracking heuristic included in heuristicblocking.js
```

Comment:
Can you set up your editor to read in our [.editorconfig](https://github.com/EFForg/privacybadger/blob/master/.editorconfig)? Python files should use four spaces instead of tabs.